src/components/Counter/Counter.jsx

```diff
@@ -1,55 +1,66 @@
-import { useEffect, useState } from 'react'
+import { useReducer, useEffect } from 'react'
 
-const pinkRGB = `rgb(236, 72, 153)`
+const colors = {
+  pink: 'rgb(236, 72, 153)',
+  green: 'rgb(52, 211, 153)',
+  red: 'rgb(239, 68, 68)',
+}
+
+const initialCountColor = { count: 0, currentColor: colors.pink }
+
+function reduce(state, action) {
+  // console.log('state', state)
+  // console.log('action', action)
+  switch (action.type) {
+    case 'increment':
+      return { count: state.count + 1, currentColor: state.currentColor }
+
+    case 'decrement':
+      return { count: state.count - 1, currentColor: state.currentColor }
+    case 'reset':
+      return { count: 0, currentColor: state.currentColor }
+    case 'change':
+      return { count: state.count, currentColor: action.payload.color }
```

Comment:
This is a great way to handle all three color changing options, great job Kamie!